### ISSUE TITLE
Use once_cell::Lazy instead of std::sync::Once

### ISSUE THREAD

**Issue by sdroege** [gtk-rs/glib#649]
See summary. Simpler and less easy to get wrong.

**Comment by GuillaumeGomez** [comment]
I was sure we were already using it...

**Comment by sdroege** [comment]
`git grep Once src` :P

**Comment by GuillaumeGomez** [comment]
Your knowledge is too huge for me to handle. :p

**Comment by sdroege** [comment]
Requires re-exporting once_cell from glib though, it's used by a macro. So maybe something for once this is merged into std :)
